cmd/copy.go

```diff
@@ -1166,6 +1373,12 @@ func init() {
 	cpCmd.PersistentFlags().BoolVar(&raw.withSnapshots, "with-snapshots", false, "include the snapshots. Only valid when the source is blobs.")
 	cpCmd.PersistentFlags().StringVar(&raw.legacyInclude, "include", "", "only include these files when copying. "+
 		"Support use of *. Files should be separated with ';'.")
+	cpCmd.PersistentFlags().StringVar(&raw.includePath, "include-path", "", "only include these paths when copying. "+
+		"Supports use of * considering the relative path of items (from the root of the search.) ex. myFolder/*.txt\n;*/subDirName/*.pdf"+
+		"Note: This considers the _entire_ relative path, file name included.")
+	cpCmd.PersistentFlags().StringVar(&raw.excludePath, "exclude-path", "", "only exclude these paths when copying. "+
+		"Supports use of * considering the relative path of items (from the root of the search.) ex. myFolder/*.txt\n;*/subDirName/*.pdf"+
```

Comment:
The new line \n might make the help message look weird with a different resolution (on other OS where it can be adjusted easily), I suggest removing it.

Reply:
Will do.